aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -678,41 +678,47 @@ Tensor& polygamma_out(int64_t n, const Tensor& self, Tensor& result) {
   return result;
 }
 
+namespace {
+constexpr double HALF = 0.5;
+constexpr double QUARTER = 0.25;
+}
+
 static inline void mvlgamma_check(const Tensor& self, int64_t p) {
   TORCH_CHECK(at::isFloatingType(self.scalar_type()),
               "mvlgamma is not implemented for ", self.scalar_type());
-  TORCH_CHECK((self > 0.5f * (p - 1)).all().item<bool>(),
+  TORCH_CHECK((self > HALF * (p - 1)).all().item<bool>(),
               "All elements must be greater than (p-1)/2");
   TORCH_CHECK(p >= 1, "p has to be greater than or equal to 1");
 }
 
 Tensor mvlgamma(const Tensor& self, int64_t p) {
   mvlgamma_check(self, p);
   Tensor args = native::arange(
-      -p / 2. + 0.5,
-      0.5,
-      0.5,
+      -p * HALF + HALF,
+      HALF,
+      HALF,
       optTypeMetaToScalarType(self.options().dtype_opt()),
       self.options().layout_opt(),
       self.options().device_opt(),
       self.options().pinned_memory_opt());
   args = args.add(self.unsqueeze(-1));
-  return args.lgamma_().sum(-1).add_(p * (p - 1) * std::log(c10::pi<double>) / 4.);
+  const auto p2_sub_p = static_cast<double>(p * (p -1 ));
+  return args.lgamma_().sum(-1).add_(p2_sub_p * std::log(c10::pi<double>) * QUARTER);
 }
 
 Tensor& mvlgamma_(Tensor& self, int64_t p) {
   mvlgamma_check(self, p);
-  auto dtype_opt = self.options().dtype_opt();
   Tensor args = native::arange(
-      -p / 2. + 0.5,
-      0.5,
-      0.5,
+      -p *HALF  + HALF,
+      HALF,
+      HALF,
       optTypeMetaToScalarType(self.options().dtype_opt()),
       self.options().layout_opt(),
       self.options().device_opt(),
       self.options().pinned_memory_opt());
   args = args.add(self.unsqueeze(-1));
-  return self.copy_(args.lgamma_().sum(-1).add_(p * (p - 1) * std::log(c10::pi<double>) / 4.));
+  const auto p2_sub_p = static_cast<double>(p * (p -1 ));
```

Comment:
minor formatting nit
```suggestion
  const auto p2_sub_p = static_cast<double>(p * (p - 1));
```

Reply:
Updated. It should have been detected by `clang-format`.